pkg/rfc2317/arpa_test.go

```diff
@@ -0,0 +1,94 @@
+package rfc2317
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestCidrToInAddr(t *testing.T) {
+	var tests = []struct {
+		in      string
+		isError bool
```

Comment:
I prefer having this be a `string`, with the test asserting that the returned error has the specified message.

Reply:
Fixed